tictactoe/tictactoe.js

```diff
@@ -0,0 +1,94 @@
+var origBoard;
```

Comment:
`var` should probably be `let`, but before you make the change, read about the differences and make a wise decision...